src/cdk/constructs/CloudFrontDistribution.ts

```diff
@@ -82,6 +98,13 @@ export class CloudFrontDistribution extends Construct {
           ],
           cachePolicy: splitCachePolicy
         },
+        '/_next/image*': {
+          origin: new origins.HttpOrigin(ebAppDomain, {
```

Comment:
nit: maybe to create it in line 73 next to the s3 origin creation

Reply:
updated